src/bootstrap/lib.rs

```diff
@@ -346,6 +348,7 @@ impl Build {
 
     /// Executes the entire build, as configured by the flags and configuration.
     pub fn build(&mut self) {
+        println!("{}", self.beta_prerelease_version());
```

Comment:
This is not essential, but maybe remove this `println!`?

Reply:
Er sorry yeah that was just debugging